### DISCUSSION TITLE
Possible to include loss variance in objective?

### DISCUSSION THREAD

**Discussion by syedzayyan** [optuna/optuna#4880] [Q&A]
Hello!

I am doing 10-fold cross-validation inside the objective function and I was wondering if I could include loss of the variance like I can with hyperopt.

Thanks!

**Comment by nzw0301** [comment]
it cannot be used for the objective, but it can be used to terminate optimisation by using the terminator feature https://optuna.readthedocs.io/en/stable/reference/terminator.html.
